content/sensu-core/2.0/reference/backend.md

```diff
@@ -0,0 +1,585 @@
+---
+title: "Backend reference"
+linkTitle: "Backend"
+description: "Reference documentation for the Sensu backend"
+weight: 1
+version: "2.0"
+product: "Sensu Core"
+platformContent: false 
+menu:
+  sensu-core-2.0:
+    parent: reference
+---
+
+- [Installation][1]
+- [Operation](#operation)
+  - [Starting the service](#starting-the-service)
+  - [Stopping the service](#stopping-the-service)
+  - [Restarting the service](#restarting-the-service)
+  - [Enabling on boot](#enabling-on-boot)
+  - [Getting service status](#getting-service-status)
+  - [Getting service version](#getting-service-version)
+  - [Getting help](#getting-help)
+- [Configuration](#configuration)
+  - [General configuration](#general-configuration-flags)
+  - [Agent communication configuration](#agent-communication-configuration-flags)
+  - [Security configuration](#security-configuration-flags)
+  - [Dashboard configuration](#dashboard-configuration-flags)
+  - [Datastore and cluster configuration](#datastore-and-cluster-configuration-flags)
+
+## How does the backend work?
+
+The Sensu backend is a service that manages check requests and event data.
+You can run the backend as a standalone service, but running a cluster of backends makes Sensu more highly available, reliable, and durable.
```

Comment:
Probably a nitpick, but the mention of clustering feels out of place. Possibly a link here (there is one a bit further down) or not mentioning clustering until later?